src/components/Progress.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import ProgressBar from 'react-bootstrap/ProgressBar';
+
+class Progress extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {};
+	}
+	render() {
+		return (
+			<section>
+				<p>How the game helps your child’s progress</p>
+				<div>
+					<div>
+						<p>Phisical</p>
+						<p>Level 1</p>
+					</div>
+					<ProgressBar variant="success" now={40} />
```

Comment:
Hi Ioana, wouldn't it be cool if, instead of a number, 40, 20 or whatever, we can use a random number so it changes everytime we refresh the page as if we were observing the evolution of a different child? Take a look at https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Objetos_globales/Math/random

Reply:
@AntonellaM I added a function for the progress bars please review it and let me know if it seems ok to you?